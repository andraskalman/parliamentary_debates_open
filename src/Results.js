import React from 'react';

const Results = ({ results }) => results.hits.map(result => (
  <p key={ result._id }>{ result._source.text }</p>
))


export default Results
