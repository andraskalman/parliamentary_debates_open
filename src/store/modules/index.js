import { combineReducers } from 'redux'
import { routerReducer } from 'react-router-redux'
import search from './search'
import help from './help'

export default combineReducers({
  routing: routerReducer,
  search,
  help
})
