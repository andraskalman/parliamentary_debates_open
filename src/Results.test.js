import React from 'react';
import ReactDOM from 'react-dom';
import Results from './Results';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux'
import store, { history } from './store'
import { ConnectedRouter } from 'react-router-redux'


it('renders correctly', () => {
  const tree = renderer
    .create(<Results results={{"hits": {"hits": [
            {"highlight": {"text": "a"}, "_source": {"text": "Foo"}, "_id": 3},
            {"highlight": {"text": "b"}, "_source": {"text": "Bar"}, "_id": 2}
        ]}}} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
})
