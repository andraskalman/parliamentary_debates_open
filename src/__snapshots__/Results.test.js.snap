// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <p>
    Foo
</p>,
  <p>
    Bar
</p>,
]
`;
